etc/scripts/get-stack.sh

```diff
@@ -328,6 +328,31 @@ do_alpine_install() {
   fi
 }
 
+# KaOS distro install
+
+do_kaos_install() {
+    while true; do
+     printf "For stack to work in KaOS is it fundamental to install libtinfo.\nDo you like to do that?" 
+     read -p " y or n: " yn
+    case $yn in
+        [Yy]*)  printf "\e[1;31m Do not edit the PKGBUILD\e[0m " && kcp -u && kcp -i libtinfo; break;;
+        [Nn]*)  break;;
+        *) printf "Please answer with y or n. ";;
+      esac
+    done
+
+    while true; do
+      echo
+      printf  "To be detectable by IDEs and other applications is it recommended to symlink \e[34mGHC\e[0m to the PATH.\nDo you like to do that?" 
+      read -p " y or n: " yn
+    case $yn in
+        [Yy]*)  sudo ln -s /home/$USER/.stack/programs/x86_64-linux/ghc-tinfo6-8.8.4/bin/ghc /usr/bin; break;;
```

Comment:
This line can only work when we soft code the version

Reply:
I don't think install scripts should ever do this. `/usr/bin` is controlled by your distros package manager.

Also note that there is a `sudocmd` function:

https://github.com/commercialhaskell/stack/blob/708b516a8d34cc42b6c533d161b18b31f913c3d6/etc/scripts/get-stack.sh#L138

Reply:
So you suggest a different dir for the PATH?
And using sudocmd instead sudo?

Reply:
My opinion is to not do anything here. Global GHC installations is probably not in the scope the install script. IDEs like HLS work with stack just fine without exposing a global GHC.

Reply:
its exactly what this script does for Debian and Co, no?

Reply:
And vscode does not

Reply:
> its exactly what this script does for Debian and Co, no?

I don't think so.